src/gantry/widgets.py

```
@@ -461,9 +466,225 @@ def _build_text(self) -> str:
+_PRESETS_FILE = Path.home() / ".config" / "gantry" / "filter_presets.json"
+
+
+def _load_filter_presets() -> Dict[str, str]:
+    """Load saved filter presets from disk."""
+    try:
+        if _PRESETS_FILE.exists():
+            data = json.loads(_PRESETS_FILE.read_text())
+            if isinstance(data, dict):
+                return data
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)